pages/common/mysqld.md

```diff
@@ -0,0 +1,19 @@
+# mysqld
+
+> The MySQL Server.
```

Comment:
Perhaps a more descriptive description would help here? Perhaps add a short sentence explaining what MySQL is.